src/NServiceBus.Testing.Tests/ApprovalFiles/APIApprovals.Approve.approved.txt

```diff
@@ -315,6 +315,7 @@ namespace NServiceBus.Testing
     {
         public TestableSubscribeContext() { }
         public System.Type EventType { get; set; }
+        public System.Type[] EventTypes { get; set; }
```

Comment:
@timbussmann just added this to make it compile, anything else needed here from a transport seam change perspective?

Reply:
`EventType` should be removed @andreasohlund

Reply:
Its still on ISubscribeContext so I have to keep it?

https://github.com/Particular/NServiceBus/blob/master/src/NServiceBus.Core/obsoletes-v8.cs#L1372

Reply:
ah yes, forgot that it is still there due to the obsolete 🤦 

> https://github.com/Particular/NServiceBus/blob/master/src/NServiceBus.Core/obsoletes-v8.cs#L1372

That get should probably throw an exception I think?